packages/orbit-components/src/OrbitProvider/index.tsx

```diff
@@ -22,6 +23,20 @@ import type { Props } from "./types";
  *
  */
 const OrbitProvider = ({ theme, children, useId }: Props) => {
+  React.useEffect(() => {
+    const ID = "orbit-theme-css-vars";
+    const style = document.createElement("style");
+    style.setAttribute("id", ID);
+    style.innerText = tokensToCssVars({ tokens: theme.orbit, cssClass: ":root" }).replace(
+      /\s/g,
+      "",
```

Comment:
I would prefer this replace to be inside the function, and not here, if possible.